packages/backend/src/services/OAuthService/OAuthService.ts

```diff
@@ -83,4 +84,32 @@ export class OAuthService extends BaseService {
         if (refreshToken) return this.oauthModel.revokeToken(refreshToken);
         return false;
     }
+
+    public async registerClient({
```

Comment:
Added new support for registering clients 

It should not be a requirement for MCP inspector, because of this https://github.com/modelcontextprotocol/inspector/pull/345 but I didn't manage to make it work 

<img width="558" height="367" alt="Screenshot from 2025-07-30 11-15-47" src="https://github.com/user-attachments/assets/c2cd3718-8c3f-4d54-9f60-96526d212fc6" />